Sources/MapboxCoreNavigation/PassiveLocationDataSource.swift

```diff
@@ -100,16 +93,16 @@ open class PassiveLocationDataSource: NSObject {
             tilesURL.appendPathComponent(bundleIdentifier, isDirectory: true)
         }
         tilesURL.appendPathComponent(".mapbox", isDirectory: true)
-        tilesURL.appendPathComponent(tilesVersion, isDirectory: true)
         // Tiles with different versions shouldn't be mixed, it may cause inappropriate Navigator's behaviour
         try FileManager.default.createDirectory(at: tilesURL, withIntermediateDirectories: true, attributes: nil)
         configureNavigator(withURL: tilesURL, tilesVersion: tilesVersion)
     }
 
-    func configureNavigator(withURL tilesURL: URL, tilesVersion: String) {
-        let endpointConfig = TileEndpointConfiguration(directions: directions, tilesVersion: tilesVersion)
+    func configureNavigator(withURL tilesURL: URL, tilesVersion: String?) {
+        let endpointConfig = TileEndpointConfiguration(directions: directions, tilesVersion: tilesVersion, minimumDaysToPersistVersion: nil)
```

Comment:
To limit the scope of this PR and derisk the v1.3.0 release, I avoided exposing the new configuration options publicly. But it should be pretty straightforward to do so. The only holdup is understanding the options well enough to document them.